src/user/importexport-app.md

```diff
@@ -251,18 +250,16 @@ recommended format and can also be used to import associated geometries
 2.  Click on the **Import** button which will upload the file and start the
     importing process.
 
-### Tracked Entity Instances Import { #tei_import }
+### Tracked Entities Import { #tei_import }
 
-Tracked Entity Instances Import can be accessed from the sidebar by
-clicking on TEI Import.
+Tracked Entities Import can be accessed from the sidebar by
```

Comment:
I think we should write proper sentences with https://www.grammarly.com/blog/capitalization-rules.

Reply:
I did not know this guide but assume its for UI only and not docs. I find this

> There are some terms in DHIS2, like organisation unit, that have adopted other spellings. In those cases, use the DHIS2 established spelling.

leads to odd sentences in documentation.